python/triton/language/libdevice.py

```diff
@@ -0,0 +1,1353 @@
+
+
+@extern.extern
+def lgamma(arg0, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, ], {(core.dtype("fp32"),): (
+        "__nv_lgammaf", core.dtype("fp32")), (core.dtype("fp64"),): ("__nv_lgamma", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def ldexp(arg0, arg1, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, arg1, ], {(core.dtype("fp32"), core.dtype("int32"),): (
+        "__nv_ldexpf", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("int32"),): ("__nv_ldexp", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def scalbn(arg0, arg1, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, arg1, ], {(core.dtype("fp32"), core.dtype("int32"),): (
+        "__nv_scalbnf", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("int32"),): ("__nv_scalbn", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def fmod(arg0, arg1, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, arg1, ], {(core.dtype("fp32"), core.dtype("fp32"),): (
+        "__nv_fmodf", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("fp64"),): ("__nv_fmod", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def remainder(arg0, arg1, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, arg1, ], {(core.dtype("fp32"), core.dtype("fp32"),): (
+        "__nv_remainderf", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("fp64"),): ("__nv_remainder", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def fma(arg0, arg1, arg2, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, arg1, arg2, ], {(core.dtype("fp32"), core.dtype("fp32"), core.dtype("fp32"),): (
+        "__nv_fmaf", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("fp64"), core.dtype("fp64"),): ("__nv_fma", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def pow(arg0, arg1, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, arg1, ], {(core.dtype("fp32"), core.dtype("int32"),): ("__nv_powif", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("int32"),): (
+        "__nv_powi", core.dtype("fp64")), (core.dtype("fp32"), core.dtype("fp32"),): ("__nv_powf", core.dtype("fp32")), (core.dtype("fp64"), core.dtype("fp64"),): ("__nv_pow", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def tgamma(arg0, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, ], {(core.dtype("fp32"),): (
+        "__nv_tgammaf", core.dtype("fp32")), (core.dtype("fp64"),): ("__nv_tgamma", core.dtype("fp64")), }, _builder)
+
+
+@extern.extern
+def round(arg0, _builder=None):
+    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, ], {(core.dtype("fp32"),): ("__nv_roundf", core.dtype("fp32")), (core.dtype(
+        "fp64"),): ("__nv_round", core.dtype("fp64")), (core.dtype("fp64"),): ("__nv_llround", core.dtype("int64")), }, _builder)
```

Comment:
It looks like there are duplicate entries here:
```
(core.dtype("fp64"),): ("__nv_round", ...
(core.dtype("fp64"),): ("__nv_llround", ...
```

Reply:
Fixed